anya/roman_numeral.rb

```diff
@@ -0,0 +1,36 @@
+class RomanNumeral
+  ROMAN_NUM = {
+              'M' => 1000,
+              'D' => 500,
+              'C' => 100,
+              'L' => 50,
+              'X' => 10,
+              'V' => 5,
+              'I' => 1
+            }
+  ROMAN_NUM.default = 0
```

Comment:
We should update the behavior here to return an MT string or something as there is no way to express the concept of `zero` in roman numerals right?